metron-interface/metron-config/e2e/sensor-config/sensor-config.po.ts

```diff
@@ -0,0 +1,243 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { browser, element, by, protractor } from 'protractor/globals';
+import {changeURL, waitForElementVisibility, waitForElementInVisibility, waitForElementPresence} from '../utils/e2e_util';
+
+export class SensorConfigPage {
+
+  _getEnrichmentsMultipleInput() {
+    return element.all(by.css('.config.container')).all(by.css('metron-config-multiple-input')).get(1).all(by.css('select')).last();
+  }
+
+  _getThreatIntelMultipleInput() {
+    return element.all(by.css('.config.container')).all(by.css('metron-config-multiple-input')).last().all(by.css('select')).last();
+  }
+
+  _getTransformationMultipleInput() {
+    return element.all(by.css('.config.container')).all(by.css('metron-config-multiple-input')).first().all(by.css('select')).last();
+  }
+
+  clickAddButton() {
+    changeURL(browser.baseUrl + '/sensors');
+    let addButton = element(by.css('.metron-add-button.hexa-button'));
+    return waitForElementPresence(addButton).then(() => {
+      addButton.click();
+    });
+  }
+
+  clickAddThreatTriageRule() {
+    let addThreatTriageButton = element(by.css('metron-config-sensor-threat-triage .add-button'));
+    return waitForElementVisibility(addThreatTriageButton).then(() => {
+      return addThreatTriageButton.click();
+    });
+  }
+
+  clickGrokStatement() {
```

Comment:
This seems surprisingly sensor specific given the generality of the other calls. Are we presuming too much about the type of sensor here?